src/types/UserModerationLogEntry.ts

```diff
@@ -0,0 +1,5 @@
+//standardize the moderation log type by using enum instead of custom string.
+export enum UserModerationLogEntryType {
+  Strike = 'Strike',
+  Mute = 'Mute',
```

Comment:
for string value, we save it into database, please keep the lowercase. e.g. `'strike'` and `'mute'`